core/src/main/java/io/paradoxical/cassieq/admin/resources/api/v1/PermissionsResource.java

```diff
@@ -93,7 +94,8 @@ public Response generateAuthUrl(@NotNull GetAuthQueryParamsRequest request) thro
                         request.getAccountName(),
                         authorizationLevels,
                         request.getStartTime(),
-                        request.getEndTime());
+                        request.getEndTime(),
```

Comment:
hey we need a good way to keep this and the query string builder below in sync. Lets think about a way to do this